### ISSUE TITLE
Jojko a MiniMax fix

### ISSUE THREAD

**Issue by Jakubicek222** [cache-sk/plugin.video.freeview.sk#7]
@cache-sk 
Prosím fixni Jojka a MiniMax
Tu máš linky:
Jojko: https://st01-1.iptv.joj.sk/104-tv-pc.m3u8
MiniMax: https://hls1eu.sweet.tv/stream/ar8ktr34cnkbfcfgq3ugx2cxqg9hfijdwekvxb8ykab4wrcmhwxhwqsm3wqb2cbkwakp24ctgdz44ptu6gu73z75ziu6rb7s3kq2bgbki43rdfb6dacanaa7wenjf5bqkhfhfdwf.m3u8
Oba sú funkčné a z oficiálnych webov.

**Comment by cache-sk** [comment]
Jojka pridam (a mozno ja dalsie), kedze je to z joj play (aj ked platene), ale MiniMax je zjavne z iptv sluzby sweet.tv, takze toho nie.

Pokusim sa co najskor.

Plugin umoznuje ulozit playlist aj mimo a mozes si ho upravit sam a popridavat si co chces, pripadne vyuzit nejaky merge plugin vklude.

**Comment by Jakubicek222** [comment]
Pošleš mi prosím link Novy Lady na mail: jakubfojtik762@gmail.com?

**Comment by Jakubicek222** [comment]
Originalne linky chces? Kludne mam Discovery od vydavatela akuratne treba prepnut stopu.
Discovery Channel: http://91.237.170.202:11000/play/11351

**Comment by Jakubicek222** [comment]
Nejake originalne detske stanice mam oficialne akuratne ze tym musis nejakym sposobom ponechat prepnutu zvukovu stopu, pretoze original je polsky. Reklamy by tiez boli polsky.

**Comment by Jakubicek222** [comment]
Podarilo sa mi z Wayback Machine vybrať stream z JimJam.cz (stránka už neexistuje) a stále ho uchovávajú v 3 jazykových stopách (česky, anglicky, maďarsky). Buď to beží na pe.jimjam.tv alebo ja neviem. Každopádne som tento link skrátil, pretože bol neuveriteľne dlhý.
JimJam: http://bit.ly/jimjam_streams-online

**Comment by Jakubicek222** [comment]
![image](https://user-images.githubusercontent.com/106910269/204146246-a3f42368-d6ed-47f6-8eef-efa321620888.png)
![image](https://user-images.githubusercontent.com/106910269/204146282-4bf9999e-15be-4725-89ba-30176aa3ac4b.png)

**Comment by cache-sk** [comment]
Nie, dalsie streamy tam nedam, toto nieje nejaka nelegalna iptv, vsetky streamy, co su v zozname, su streamy samotnych vysielatelov (televizii), nie streamy ziskane z nejakych platenych iptv, ci nelegalne retransmisie na pofidernych ip adresach a portoch (ten jimjam je kradnuty stream zo skylinku napr. a nie ze ma jimjam na svojej stranke player kde si to mozes spustit)

Link na Nova Lady ti bohuzial nedam, nieje staticky, ziskavany je podobne ako aj pri inych streamoch, vzdy priamo zo stranky vysielatela, kde je sucastou URL aj session token, cize ten stream je prehratelny len vtedy a len u toho, komu je vystaveny. Koli tomu vznikol tento plugin, nie je to zoznam statickych m3u liniek, v playliste su vsetko spatne linky na plugin, kde sa vzdy spusti nejaky kod.

Pridane su vsetky stanice streamovane na joj play.

**Comment by cache-sk** [comment]
Ono to interne pouziva m3u playlist, ale v tom playliste nie su priame linky na streamy, ale spatne linky na plugin, ako som uz pisal.

Vies pouzit IPTV Merge od SlyGuy na mergovanie m3u playlistov, ked otvoris doplnok, tak vie aj ulozit niekam m3u playlist (teda... ked som to robil to fungovalo, odvtedy som neskusal), mas ten playlist dokoca aj v [zdrojovych kodoch](https://github.com/cache-sk/plugin.video.freeview.sk/blob/master/resources/playlist.m3u), ale nebude fungovat mimo Kodi, kde je nainstalovany tento plugin
